lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.scss

```diff
@@ -36,7 +36,9 @@
     }
 
     #{$mat-form-field-icon-suffix} {
-        align-self: baseline;
+        position: absolute;
+        right: 1px;
+        bottom: 7px;
```

Comment:
Could you please test this change with increased font size? So we can be sure if this is going to work also with different font sizes

Reply:
@BSekula which font size do you mean? There are multiple elements with different font sizes.
Increasing the font size of the input looks like this:
![image](https://github.com/user-attachments/assets/905019e8-26c3-499d-99b2-68efa4db32c0)

Reply:
Looks Ok